pkg/proto/sqlflow.proto

```diff
@@ -63,15 +63,11 @@ message FetchRequest {
 }
 
 message FetchResponse {
-    message Logs {  //  depreated
-        repeated string content = 1;
-    }
     message Responses {
         repeated Response response = 1;
     }
     FetchRequest updated_fetch_since = 1;
     bool eof = 2;
-    Logs logs = 3;  //  depreated
     Responses responses = 4;
```

Comment:
I keep `responses = 4` instead of `3` because `jsqlflow` also uses `4`. https://github.com/sql-machine-learning/jsqlflow/blob/689cf5a6786959dd1f95b756a0b29ea61091c99c/src/main/proto/sqlflow.proto#L71

@weiguoz, shall we move `jsqlflow` repo to the `sqlflow` repo so that we can make sure their proto files are consistent?

Reply:
Add a comment here to explain why no `3` here?

Reply:
> I keep `responses = 4` instead of `3` because `jsqlflow` also uses `4`. https://github.com/sql-machine-learning/jsqlflow/blob/689cf5a6786959dd1f95b756a0b29ea61091c99c/src/main/proto/sqlflow.proto#L71
> 
> @weiguoz, shall we move `jsqlflow` repo to the `sqlflow` repo so that we can make sure their proto files are consistent?

I'm not sure if we should do this, for: 
- Even move the `jsqlflow` repo under `sqlflow`, we still need to copy the `.proto` file under the Java project.
- The [version of `jsqlflow` library](https://github.com/sql-machine-learning/jsqlflow/blob/689cf5a6786959dd1f95b756a0b29ea61091c99c/pom.xml#L7-L10) should follow the `.proto`. If the `.proto` is updated, but we didn't upgrade `jsqlflow` version, the library user would be confused.

Reply:
> I keep `responses = 4` instead of `3` because `jsqlflow` also uses `4`. https://github.com/sql-machine-learning/jsqlflow/blob/689cf5a6786959dd1f95b756a0b29ea61091c99c/src/main/proto/sqlflow.proto#L71
> 
> @weiguoz, shall we move `jsqlflow` repo to the `sqlflow` repo so that we can make sure their proto files are consistent?

Hi @tonyyang-svail ,
I would suggest adding the `sqlflow` repo as a submodule of `jsqlflow`, so we have the chance to control the version of `sqlflow.proto`.
Please take a look at this [PR](https://github.com/sql-machine-learning/jsqlflow/pull/31).